src/ethereum_test_tools/common/types.py

```diff
@@ -1116,6 +1116,22 @@ def serialize_transactions(input_txs: List[Transaction] | None) -> bytes:
     return eth_rlp.encode(transaction_list_to_serializable_list(input_txs))
 
 
+def blob_versioned_hashes_from_transactions(input_txs: List[Transaction] | None) -> List[bytes]:
+    """
+    Gets a list of ordered blob versioned hashes from a list of transactions.
+    """
+    versioned_hashes: List[bytes] = []
+
+    if input_txs is None:
+        return versioned_hashes
+
+    for tx in input_txs:
+        if tx.blob_versioned_hashes is not None:
```

Comment:
I think we need to include the tx type check, maybe we could add it here:
```suggestion
        if tx.blob_versioned_hashes is not None and tx.type >= 3:
```

Reply:
Thanks! I modified the field name from `type` to `ty` and used `== 3` instead of `>= 3` because we don't know yet if all new types of transactions could contain blobs.